### DISCUSSION TITLE
I

### DISCUSSION THREAD

**Discussion by boardtc** [Devolutions/UniGetUI#1581] [Q&A]
I only discovered this after the newsletter mentioned.  I right-clicked and ran with admin privileges but it keeps popping up looking for  permission for cmd logging:
![image](https://github.com/marticliment/WingetUI/assets/214808/309f9c2c-1924-4b71-9f28-d925883473ec)
Is there a way to stop this?

**Comment by marticliment** [comment]
Hello,

Your IT department has set a policy on your pc that will warn you every time cmd.exe is launched. You will need to contact them in order to change this policy, since this issue cannot be fixed from WingetUI
